src/Avalonia.Themes.Fluent/Controls/ProgressBar.xaml

```diff
@@ -32,7 +36,17 @@
                 <Border x:Name="IndeterminateProgressBarIndicator2" CornerRadius="{TemplateBinding CornerRadius}" Margin="{TemplateBinding Padding}" Background="{TemplateBinding Foreground}" />
               </Panel>
               <LayoutTransformControl x:Name="PART_LayoutTransformControl" HorizontalAlignment="Center" VerticalAlignment="Center" IsVisible="{TemplateBinding ShowProgressText}">
-                <TextBlock Foreground="{DynamicResource SystemControlForegroundBaseHighBrush}" Text="{ReflectionBinding Value, RelativeSource={RelativeSource TemplatedParent}, StringFormat={}{0:0}%}" />
+                <TextBlock Foreground="{DynamicResource SystemControlForegroundBaseHighBrush}">
```

Comment:
Default theme should also be updated, since ProgressTextFormat can be used everywhere,

Reply:
Yes, will do with the docs and stuff. Are you happy enough with the implementation currently for me to move on to doing all that?

Reply:
Yes, looks good to me. Thanks!